StoreAPI/Pages/About.cshtml.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace StoreAPI.Pages
+{
+    public class AboutModel : PageModel

```

Comment:
We do not implement any UI parts, so Razor Pages code and views should not be in the API project.